src/shared/shared.color-utils.ts

```diff
@@ -1,3 +1,7 @@
+import { normal } from 'color-blend';
```

Comment:
nit: not a fan of she `shared/shared.color-utils.ts` also mixing the `.` and `-` plays funny with my brain :D